pythnet/pythnet_sdk/src/test_utils/mod.rs

```diff
@@ -155,12 +178,43 @@ pub fn create_vaa_from_payload(
     emitter_chain: Chain,
     sequence: u64,
 ) -> Vaa<Box<RawMessage>> {
-    let vaa: Vaa<Box<RawMessage>> = Vaa {
-        emitter_chain: emitter_chain,
-        emitter_address: emitter_address,
+    let guardians = create_dummy_guardians();
+
+    let body: Body<Box<RawMessage>> = Body {
+        emitter_chain,
+        emitter_address,
         sequence,
         payload: <Box<RawMessage>>::from(payload.to_vec()),
         ..Default::default()
     };
-    vaa
+
+    let digest = libsecp256k1Message::parse_slice(&body.digest().unwrap().secp256k_hash).unwrap();
+
+    let signatures: Vec<(Signature, RecoveryId)> = guardians[0..NUM_SIGNATURES]
```

Comment:
maybe the number of signatures should be an argument to the function

Reply:
also do you want it to be the case that it's always the first N guardians who signed? are there possible tests where you want a random subset of guardians or something?

I would think about the range of configurability you need downstream and then try to build that in to the test methods here. 

Another similar question would be whether you need the ability to define & work with different guardian sets in the tests. In that case, you may want to make the signing guardian keys an argument to this function.

Reply:
Ok so I switches to having random subsets of guardians